tests/unit/components/ObsDetails/ObsDetails.test.js

```diff
@@ -63,17 +64,27 @@ jest.mock( "../../../../src/components/LoginSignUp/AuthenticationService", ( ) =
 
 jest.mock( "components/ObsDetails/AddCommentModal" );
 
+jest.mock( "sharedHooks/useIsOnline" );
+
+const mockLatLng = {
+  latitude: "91",
+  longitude: "-121"
+};
+
+jest.mock( "sharedHooks/useUserLocation", ( ) => ( {
```

Comment:
This is a question: Why you don't use the user factory for this type of data?

Reply:
We could, I guess. In this case, `useUserLocation` is fetching geolocation from the user's device, so in my mind this doesn't really have anything to do with the realm User object. I'd be more inclined to make a `DeviceLocation.js` factory to mock this.